source/application/VESPERS/VESPERSAppController.cpp

```diff
@@ -168,8 +169,8 @@ bool VESPERSAppController::startup()
 			firstRun.storeToDb(AMDatabase::database("user"));
 		}
 
-		if (!ensureProgramStructure())
-			return false;
+//		if (!ensureProgramStructure())
+//			return false;
```

Comment:
Was this so you could run it on your laptop? Will it need to be re-added before merge?